kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/sidePanels/LessonResourceSelection/subPages/ManageSelectedResources.vue

```diff
@@ -0,0 +1,263 @@
+<template>
```

Comment:
This isnt actually a completely new file, this was moved from https://github.com/learningequality/kolibri/pull/12996/files#diff-31fd90e627499b35b0a5e2f15e2b3fb570cb32543799343fd4549daf91bbe2a4

Reply:
Did `git mv` not work to retain the blame?

Reply:
Ok, after some risky git operations, I was able to do it, it seems in cases like this, where the diff will be super big after the move of the file, its better to isolate the file movement in a single commit, and then in a different commit, we can start modifying the file. This way git is forced to take the new diff as diff of a renamed file, comparing it to the same file in the previous commit. Although github in the PR changed files doesnt show this file as a rename because of its large diff, if we check the file, we can now see the proper blame.

![image](https://github.com/user-attachments/assets/ba08e757-9f80-47e8-99d4-0d3aaf60d184)